library/common/BUILD

```diff
@@ -4,6 +4,19 @@ licenses(["notice"])  # Apache 2
 
 envoy_package()
 
+envoy_cc_library(
+    name = "config_lib",
+    srcs = [
+        "certificates.inc",
```

Comment:
additional clean up: should we move these files to a subdir?

Reply:
There's significant cleanup coming in phases 2/3, I propose we defer till then since a lot of this changes.